static/js/scenesUniqueIdTagging.js

```diff
@@ -13,6 +13,7 @@ var LINES_CHANGED_LISTENER_TIMEOUT = 800;
 var SCENE_ID_KEY_ATTRIB = shared.SCENE_ID_KEY_ATTRIB;
 var SCENE_ID_PREFIX = shared.SCENE_ID_PREFIX;
 var SCENE_ID_REGEXP = shared.SCENE_ID_REGEXP;
+var SCENE_WAS_MARKED = 'scene_was_marked';
```

Comment:
Essa constante deveria estar no `shared.js` pq ela vai ser usada por quem for escutar o evento.